vue/components/organisms/pickers/pickers.vue

```diff
@@ -146,6 +157,10 @@
     vertical-align: top;
 }
 
+body.desktop .pickers .parts-container > .part.button-extra {

```

Comment:
I understand why this is here, but you're letting yourself get influenced by RIPE White's task. In this context it makes no sense, why would I want to hide these extra buttons on desktop?

It's RIPE White who must do this.